Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -92,7 +93,7 @@ public override void Update(float frameTime)
             // End DeltaV Additions
             UpdateSaturation(uid, multiplier * (float) respirator.UpdateInterval.TotalSeconds, respirator); // DeltaV: use multiplier instead of negating
 
-            if (!_mobState.IsIncapacitated(uid) && !HasComp<DebrainedComponent>(uid)) // Shitmed Change - Cannot breathe in crit or when no brain.
+            if (!_mobState.IsIncapacitated(uid) || HasComp<AffectedByCPRComponent>(uid) && !HasComp<DebrainedComponent>(uid)) // Shitmed Change - Cannot breathe in crit or when no brain. // DeltaV - Addition of CPR
```

Comment:
Probably meant to parenthesize the `||`
Currently it's `!isIncapacitated || (affectedByCPR && !debrained)`, instead of `(!isIncapacitated || affectedByCPR) && !debrained`
```suggestion
            if ((!_mobState.IsIncapacitated(uid) || HasComp<AffectedByCPRComponent>(uid)) && !HasComp<DebrainedComponent>(uid)) // Shitmed Change - Cannot breathe in crit or when no brain. // DeltaV - Addition of CPR
```

Reply:
Truthnuke